src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,15 +1,48 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Arrays;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
 
+    private static final int STORAGE_CAPACITY = 10;
+
+    private int storageFullness = 0;
```

Comment:
```suggestion
    private int storageFullness;
```
or initialise in the constructor

Reply:
done